tools/make-npm.sh

```diff
@@ -18,6 +18,7 @@ fi
 rm -rf $DES
 
 ./tools/make-nodejs.sh $DES
+./tools/make-assets.sh $DES
```

Comment:
I'm just running `./tools/make-assets.sh` for consistency. The `assets` directory is in `.npmignore`.

Reply:
A problem I see left is that the results depends on the filter lists, and these change constantly, so this will inevitable cause spurious errors -- I gather `results.json.gz` is not meant to be constantly updated? In such case maybe we should allow a second argument to `make-assets.sh` which is the commit hash to use when importing from uAssets.

Reply:
Yes, this is a problem, and this is why even in the Cliqz benchmarks we don't want to update the lists too often (maybe once every few weeks at most).

Now the problem with the commit hash is that if you have a shallow clone you don't have the history.

What about if we just copied the lists over for testing purposes every once in a while?

Reply:
> What about if we just copied the lists over for testing purposes every once in a while?

By this I mean that we keep separate copies for the tests right in the `platform/npm` directory, perhaps under something like a `test-data` subdirectory.

Reply:
Copied somewhere in `platform/npm`?

Reply:
>Copied somewhere in `platform/npm`?

Essentially instead of a `results.json.gz` we have a `test-data.tar.gz` and it includes everything, which we then unbundle as part of the build script.

Reply:
Bundle sounds good :+1:

Reply:
Done.